charts/kyuubi/templates/kyuubi-statefulset.yaml

```diff
@@ -75,6 +75,10 @@ spec:
               containerPort: {{ $frontend.port }}
             {{- end }}
             {{- end }}
+            {{- if .Values.monitoring.prometheus.enabled }}
+            - name: prometheus
+              containerPort: {{ .Values.monitoring.prometheus.port}}
```

Comment:
Please, add one space after `port` here as well. I know that it's me who suggested changes without the space., sorry for that.
```yaml
containerPort: {{ .Values.monitoring.prometheus.port }}
```